lms/djangoapps/courseware/access.py

```diff
@@ -173,51 +174,14 @@ def has_access(user, action, obj, course_key=None):
                     .format(type(obj)))
 
 
-# ================ Implementation helpers ================================
```

Comment:
I feel it is not necessary as every method holds the information(docs) explaining what is going in that. If you think it should be there I am open to add it again.
Thanks.